ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-stable-4.16-upgrade-from-stable-4.12.yaml

```diff
@@ -66,43 +71,47 @@ releases:
     release:
       architecture: amd64
       channel: candidate
-      version: "4.15"
+      version: "4.16"
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: azure-ipi-ovn-ipsec-f14
-  cron: 43 3 7,21 * *
+- as: aws-ipi-ovn-ipsec-f14
+  cron: 12 11 7,23 * *
   steps:
-    cluster_profile: azure-qe
+    cluster_profile: aws-qe
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-target
```

Comment:
I think we do not need this line in stable -> stable arm upgrade.

Reply:
I think to avoid confutation, it is better to keep this line in all configs